tests/container/image_test.py

```diff
@@ -82,31 +79,23 @@ def assertTarInfo(self, tarinfo, uid, gid, mode, isdir):
 
     def test_files_base(self):
         with TestImage('files_base') as img:
-            self.assertDigest(
```

Comment:
I'm assuming these assets are handled by file_test? Just want to confirm.

Reply:
All of these digests failed somehow. I feel asserting digests are not a good idea, because they are very sensitive to trivial changes. Because we are already asserting the number of layers and top layer, asserting digests makes these tests unnecessarily brittle.

Reply:
By asserting the hash of the container we are determining if our rules are reproducible. For now I'll LGTM this since working tests running in CI is better than ideology but finding some way to make this work would be fantastic.